x-pack/plugin/core/src/main/resources/logs-default-settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "template": {
+    "settings": {
+      "index": {
+        "lifecycle": {
+          "name": "logs-default-policy"
+        },
+        "codec": "best_compression",
+        "refresh_interval": "5s",
+        "query": {
+          "default_field": ["message"]
+        },
+        "number_of_routing_shards": "30"
```

Comment:
Do we still need this? What is the default if we don't set it?

Reply:
Number of routing shards has been removed in the default Beats templates. We have added it set only for ES 6.x. See 6.8 docs for [Split Index](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/indices-split-index.html#indices-split-index).